autoprofile/internal/sampler_scheduler.go

```diff
@@ -174,7 +174,7 @@ func (ss *SamplerScheduler) startProfiling() bool {
 	}
 	ss.samplerStart = time.Now().UnixNano()
 	ss.samplerTimeout = NewTimer(time.Duration(ss.config.MaxSpanDuration)*time.Second, 0, func() {
-		ss.Stop()
+		ss.stopSampler()
```

Comment:
With this we're not going to clear the `started` flag and stop the reporting. Is this intended?

Reply:
`Start` and `Stop` are used to start and stop the scheduler, not the sampling itself.

Reply:
Right, I see 👍